### PR TITLE
refactor(serrver/api variables)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Increase readability and improve consistency of servers/apy.py variable names

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [x] Do make sure orderportal behaves as expected

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/server/api.py [hunk 1]
```diff
@@ -145,73 +145,71 @@ def submit_order(order_type):
 @BLUEPRINT.route("/cases")
 def parse_cases():
     """Fetch cases."""
```
Comment (henrikstranneheim) [inline cg/server/api.py:147]:
```suggestion
    """Return cases."""
```

File: cg/server/api.py [hunk 2]
```diff
@@ -145,73 +145,71 @@ def submit_order(order_type):
...
-    records = db.cases(days=31)
-    count = len(records)
-    return jsonify(cases=records, total=count)
+    cases: List[Family] = db.cases(days=31)
+    count = len(cases)
+    return jsonify(cases=cases, total=count)
```
Comment (henrikstranneheim) [inline cg/server/api.py:150]:
```suggestion
    return jsonify(cases=cases, total=len(cases)
)
```

File: cg/server/api.py [hunk 3]
```diff
@@ -145,73 +145,71 @@ def submit_order(order_type):
...
 @BLUEPRINT.route("/families_in_collaboration")
 def parse_families_in_collaboration():
     """Fetch families in collaboration."""
-    order_customer: Customer = db.get_customer_by_customer_id(
-        customer_id=request.args.get("customer")
-    )
+    customer: Customer = db.get_customer_by_customer_id(customer_id=request.args.get("customer"))
     data_analysis: str = request.args.get("data_analysis")
-    families_q: Query = db.families(
+    cases_q: Query = db.families(
```
Comment (henrikstranneheim) [inline cg/server/api.py:180]:
```suggestion
    cases_query: Query = db.families(
```

**Comment by ChrOertlin** [comment]
@Vince-janv @seallard @henrikstranneheim @diitaz93 could you have a look?

File: cg/server/api.py [hunk 4]
```diff
@@ -145,73 +145,71 @@ def submit_order(order_type):
...
+    if case is None:
         return abort(http.HTTPStatus.NOT_FOUND)
-    if not g.current_user.is_admin and (case_obj.customer not in g.current_user.customers):
+    if not g.current_user.is_admin and (case.customer not in g.current_user.customers):
         return abort(http.HTTPStatus.FORBIDDEN)
 
-    data = case_obj.to_dict(links=True, analyses=True)
-    return jsonify(**data)
+    parsed_case = case.to_dict(links=True, analyses=True)
+    return jsonify(**parsed_case)
```
Comment (henrikstranneheim) [inline cg/server/api.py:201]:
```suggestion
    return jsonify(**case.to_dict(links=True, analyses=True))
```

File: cg/server/api.py [hunk 5]
```diff
@@ -312,23 +310,23 @@ def parse_flow_cells() -> Any:
...
-    return jsonify(**record.to_dict(samples=True))
+    return jsonify(**flow_cell.to_dict(samples=True))
 
 
 @BLUEPRINT.route("/analyses")
 def parse_analyses():
     """Fetch analyses."""
     if request.args.get("status") == "delivery":
-        analyses_q = db.analyses_to_deliver()
+        analyses = db.analyses_to_deliver()
```
Comment (henrikstranneheim) [inline cg/server/api.py:323]: Add type hint. Many functions to initialize  analyses here

File: cg/server/api.py [hunk 6]
```diff
@@ -312,23 +310,23 @@ def parse_flow_cells() -> Any:
...
     elif request.args.get("status") == "upload":
-        analyses_q = db.analyses_to_upload()
+        analyses = db.analyses_to_upload()
     else:
-        analyses_q = db.Analysis.query
-    data = [analysis_obj.to_dict() for analysis_obj in analyses_q.limit(30)]
-    return jsonify(analyses=data, total=analyses_q.count())
+        analyses = db.Analysis.query
+    data = [analysis_obj.to_dict() for analysis_obj in analyses.limit(30)]
```
Comment (henrikstranneheim) [inline cg/server/api.py:328]:
```suggestion
    parsed_analyses: List[dict] = [analysis_obj.to_dict() for analysis_obj in analyses.limit(30)]
```

File: cg/server/api.py [hunk 7]
```diff
@@ -389,21 +387,21 @@ def parse_current_user_information():
 @is_public
 def parse_applications():
     """Fetch application tags."""
-    query: List[Application] = db.get_applications_is_not_archived()
-    data = [record.to_dict() for record in query]
-    return jsonify(applications=data)
+    applications: List[Application] = db.get_applications_is_not_archived()
+    parsed_applications = [application.to_dict() for application in applications]
```
Comment (henrikstranneheim) [inline cg/server/api.py:391]:
```suggestion
    parsed_applications: List[dict] = [application.to_dict() for application in applications]
```

**Review by henrikstranneheim — APPROVED** [review]
Great work! A  lot cleaner 👍 !

**Comment by ChrOertlin** [comment]
Stage:

Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/hiseq.clinical
done.
Log deploy... done.
cg, version 27.1.18


Production:

Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/hiseq.clinical
done.
Log deploy... done.
cg, version 27.1.18

**Pull request merged**